docs/general/administration/hardware-acceleration/rockchip.md

```diff
@@ -177,6 +179,8 @@ Root permission is required.
 
 ### Configure With Linux Virtualization
 
+Before proceeding, please complete the [Configure on Linux Host](#configure-on-linux-host) section above.
```

Comment:
I think link to the OpenCL runtime section would be enough

Reply:
More precisely:

> Install the ARM Mali OpenCL runtime (libmali) on host:

Reply:
Don't they also need steps 2 and 3? Those look like required steps to me as they are verifying things like kernel version and configuring permissions for applications to use the GPU.

Reply:
For RK3588 the earliest BSP supports that SoC is the 5.10 LTS.

The user uid used in this section is not necessarily the "jellyfin" created in that section, and users will still have permission errors if they just copied what is provided. This is an advanced-user only document so we don't have to list everything in detail IMO. 

What's more, a lot of vendor-provided kernels enabled the rockchip hack to let every user access the rockchip hardware transcoding devices and makes even the permission configuration unnecessary. 

You can add on sentence like if you are facing permission issues, please check if the user used in the docker container has required permissions described in the previous section, and provide the link.

Reply:
The `--privileged` docker flag already avoids permission issues. If the user doesn't want to use this, they can still use `getent group video | cut -d: -f3` and `--group-add="122" \ # Change this to match your "video" host group id and remove this comment`.

Reply:
Hi,

I just wanted to say that I still had permission issues with the `--privileged` docker flag set. From what I read in the [docker documentation](https://docs.docker.com/reference/cli/docker/container/run/#privileged), the `--privileged` docker flag just allows to expose more files from the host to the container and gives more access to the container. However, the permissions of the exposed files from the host still apply so if the container user doesn't have the right UID or GID, he won't be able to access the exposed files.

In the end, I decided to follow most of the steps from the section `Configure On Linux Host` in order to create a jellyfin user and give the right permissions to the devices. Then, in docker, I just manually specified the devices, the UID and GID of the jellyfin user and the docker flag `--group-add="video"` and everything worked. I did not use the `--privileged` docker flag because I think that there's no point at both specifying the devices manually and using this flag, and it is safer not to use it.